docs/roadmap.md

```diff
@@ -36,12 +37,14 @@ During Devnet, the current stage, the team will set the foundational components
     - Narwhal and Bullshark
 - Network Governance and Economics
 - EVM Compatible application layer
-- L1 `>` L2 `>` Fleek Network Bridge
+- L1 <> L2 <> Fleek Network Bridge
```

Comment:
Otherwise, it won't build. Actually, its best to try to build the project locally to make sure

```suggestion
- L1 `<>` L2 `<>` Fleek Network Bridge
```